bp-bgt-worldmap/setup-bp-bgt-worldmap.tp2

```diff
@@ -283,7 +283,7 @@ END
 /////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\
 /////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\
 BEGIN ~BG2 style~  //Larger worldmap screen window for the worldmap menu.ui
-DESIGNATED 4
+DESIGNATED 4 LABEL ~worldmap-fullscreen-bgee~
```

Comment:
Should be `bgee2` as it is brown style, not blue from bgee

Reply:
Correct, good catch, thanks!

Reply:
BGEE style could be implemented in the future, who knows :D